apps/website/src/pages/our-projects.js

```diff
@@ -0,0 +1,701 @@
+              Strapi
+            </atoms.Typography>
+
+            <atoms.Typography
+              as="p"
+              variant="secondary"
+              size="body_lg"
+              textAlign="center"
+              className="max-w-3xl pb-8 text-gray-500"
+            >
+              <span className="font-bold">Mentor: </span>
+              <Link href="#" className="text-blue-600 underline">
+                Francis Anigbogu — Product Manager
+              </Link>
+            </atoms.Typography>
+          </div>
+        </section>
+
+        <section className="max-w-full py-24 px-4 md:px-8">
+          <div className="flex flex-col justify-center items-center">
+            <atoms.Typography
+              as="h2"
+              variant="primary"
+              size="xl4"
+              textWeight="semibold"
+              textAlign="center"
+              className="max-w-7xl pb-8"
+            >
+              How Projects & Departments Fit Together
+            </atoms.Typography>
+
+            <atoms.Typography
+              as="p"
+              variant="secondary"
+              size="body_lg"
+              textAlign="center"
+              className="max-w-3xl pb-8 text-gray-500"
+            >
+              <span className="font-bold">Projects (Product Teams): </span>
+              Where you ship real features. Teams evolve as ideas mature and
+              products grow.
+            </atoms.Typography>
+
+            <atoms.Typography
+              as="p"
+              variant="secondary"
+              size="body_lg"
+              textAlign="center"
+              className="max-w-3xl pb-8 text-gray-500"
+            >
+              <span className="font-bold">
+                Departments (Learning Communities){' '}
+              </span>
+              Your primary instruction home—peers and mentors in the same craft:
+              Design, Development, Research, Product Management. Departments run
+              role-specific sessions (critiques, PR reviews, readouts, planning)
+              and set your learning outcomes.
+            </atoms.Typography>
+
+            <atoms.Typography
+              as="p"
+              variant="secondary"
+              size="body_lg"
+              textAlign="center"
+              className="max-w-3xl pb-8 text-gray-500"
+            >
+              <span className="font-bold">
+                → See role outcomes and weekly rhythm on{' '}
+              </span>
+              <Link href="#" className="text-blue-600 underline">
+                Curriculum
+              </Link>
+            </atoms.Typography>
+
+            <atoms.Typography
+              as="p"
+              variant="secondary"
+              size="body_lg"
+              textAlign="center"
+              className="max-w-3xl pb-8 text-gray-500"
+            >
+              <span className="font-bold">Mentor: </span>
+              <Link href="#" className="text-blue-600 underline">
```

Comment:
In GitBook it's linked to the each team lead in `our-mentors-and-instructor-team` page.